apps/react-native/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@brandingbrand/fsapp": "^11.77.1",
+    "fsdev": "file:../../../local_npm_modules/fsdev",
     "@react-native-async-storage/async-storage": "^1.17.11",
     "@react-native-cookies/cookies": "^6.2.1",
     "react": "18.1.0",
```

Comment:
at the bottom of this file also recognize that there is already a babel configuration

Reply:
Preset is now included in babel.config.js